frontend/.gitignore

```diff
@@ -0,0 +1,23 @@
+# See https://help.github.com/articles/ignoring-files/ for more about ignoring files.
```

Comment:
I wonder if the other folders can be removed now since they are of historical interest only? They are already in git history for anyone who wants to look them up.